deploy/dockerephemeral/coredns-config/db.example.com-new-serial

```diff
@@ -0,0 +1,19 @@
+$ORIGIN example.com.
+@	1 IN	SOA sns.dns.icann.org. noc.dns.icann.org. (
+				2023101617 ; serial
```

Comment:
This serial number is 1 higher than the value in the http2-test file. Otherwise, this is the same as the original DB file.